src/app/components/mnoe-reviews-list/mnoe-reviews-list.coffee

```diff
@@ -31,8 +32,6 @@
           MnoeReviews.updateDescription(review).then(
             (success) ->
               reviewEdited = success.data.app_review
-              # find the no edited review
-              reviewNoEdited = _.find(scope.listOfReviews, {id: reviewEdited.id})
               # update description in dom
               reviewNoEdited.description = reviewEdited.description
```

Comment:
You can directly use review, instead of creating a reference
`review.description = reviewEdited.description`

Reply:
Hi @alexnoox ,

I am not creating a reference but changing the field description of the object that is already in the dom. Also, I can not update the whole object, the response only contains the review and not the comment within the reviews.
```
    scope.openEditModal = (review) ->
      reviewNoEdited = review
      $uibModal.open(
        templateUrl: 'app/components/mnoe-modals/comment-edit-modal.html'
        controller: 'CommentEditModal'
        resolve:
          review: angular.copy(review)
      ).result.then(
        (review) ->
          MnoeReviews.updateDescription(review).then(
            (success) ->
              reviewEdited = success.data.app_review
              # update description in dom
              reviewNoEdited.description = reviewEdited.description
            )
      )
```

Is that ok? what do you think?

Reply:
I didn't see that `review` is being copied.